examples/app/polymer/polymer.controller.js

```diff
@@ -0,0 +1,68 @@
+angular.module('makeithappen').controller('PolymerController', function() {
+    var vm = this;
+    vm.schema = {
```

Comment:
can't we load the schema and example data from a data-schema.json file just like we do on the webpage? we would reduce the duplication by a lot